src/main/scala/dpla/ingestion3/mappers/providers/NcMapping.scala

```diff
@@ -0,0 +1,190 @@
+package dpla.ingestion3.mappers.providers
+
+import dpla.ingestion3.enrichments.normalizations.filters.{DigitalSurrogateBlockList, FormatTypeValuesBlockList}
+import dpla.ingestion3.mappers.utils.{Document, XmlExtractor, XmlMapping}
+import dpla.ingestion3.messages.IngestMessageTemplates
+import dpla.ingestion3.model.DplaMapData._
+import dpla.ingestion3.model.{nameOnlyAgent, _}
+import dpla.ingestion3.utils.Utils
+import org.json4s.JValue
+import org.json4s.JsonDSL._
+
+import scala.xml._
+
+
+class NcMapping extends XmlMapping with XmlExtractor with IngestMessageTemplates {
+
+  val formatBlockList: Set[String] =
+    DigitalSurrogateBlockList.termList ++
+      FormatTypeValuesBlockList.termList
+
+  // ID minting functions
+  override def useProviderName(): Boolean = true
+
+  override def getProviderName(): String = "digitalnc"
+
+  override def originalId(implicit data: Document[NodeSeq]): ZeroToOne[String] =
+    extractString(data \ "header" \ "identifier")
+      .map(_.trim)
+
+  // SourceResource mapping
+//  override def collection(data: Document[NodeSeq]): ZeroToMany[DcmiTypeCollection] =
+//  // FIXME should pull from setSpec
+//    (data \\ "mods" \ "relatedItem")
+//      .flatMap(node => getByAttribute(node.asInstanceOf[Elem], "type", "host"))
+//      .flatMap(collection => extractStrings(collection \ "titleInfo" \ "title"))
+//      .map(nameOnlyCollection)
```

Comment:
Did you want to leave this commented-out method here?  Ok either way, just checking.

Reply:
Going to leave it commented out for now. Added some additional comments explaining why it is there for now.